src/test/java/hudson/plugins/jira/JiraChangeLogAnnotatorTest.java

```diff
@@ -65,6 +75,22 @@ public void testAnnotate() throws Exception {
         Assert.assertTrue(text.toString(false).contains(TITLE));
     }
 
+    @Test
+    public void testAnnotateWf() throws Exception {
+    	Run b = mock(Run.class);
+
+        when(b.getAction(JiraBuildAction.class)).thenReturn(new JiraBuildAction(b, Collections.singleton(new JiraIssue("DUMMY-1", TITLE))));
+
+        MarkupText text = new MarkupText("marking up DUMMY-1.");
+        JiraChangeLogAnnotator annotator = spy(new JiraChangeLogAnnotator());
+        doReturn(site).when(annotator).getSiteForProject((AbstractProject<?, ?>) Mockito.any());
+
+        annotator.annotate(b, null, text);
+
+        // make sure '$' didn't confuse the JiraChangeLogAnnotator
+        Assert.assertTrue(text.toString(false).contains(TITLE));
```

Comment:
please use form `assertThat(text.toSting(false), containsString(TITLE));` as more verbosibe in case of fail